README.md

```diff
@@ -93,7 +93,75 @@ Utilities and CLIs around the Compact compiler and builder.
   - `compact-compiler` → `packages/cli/dist/runCompiler.js`
   - `compact-builder` → `packages/cli/dist/runBuilder.js`
 
-Useful commands:
+#### Compiler CLI Options
+
+```bash
+compact-compiler [options]
+
+Options:
+  --dir <directory>   Compile specific subdirectory within src
+  --src <directory>   Source directory (default: src)
+  --out <directory>   Output directory (default: artifacts)
+  --hierarchical      Preserve source directory structure in artifacts output
+  --skip-zk           Skip zero-knowledge proof generation
+  +<version>          Use specific toolchain version (e.g., +0.26.0)
+```
+
+#### Artifact Output Structure
```

Comment:
The README additions look good and are important for CLI usage. I think we need to create a README for the cli package (oops 😅 ) and add this there. I don't think we should oversaturate the root readme...a quickstart with basic usage is okay if you agree. The proposed additions feel more comprehensive